Modules/Relationships/src/Relationships.ConsumerApi/Controllers/RelationshipsController.cs

```diff
@@ -121,6 +130,12 @@ public async Task<IActionResult> CreateRelationship(CreateRelationshipCommand re
     public async Task<IActionResult> AcceptRelationshipChange([FromRoute] RelationshipId relationshipId,
         [FromRoute] RelationshipChangeId changeId, CompleteRelationshipChangeRequest request, CancellationToken cancellationToken)
     {
+        var relationship = await _mediator.Send(new GetRelationshipQuery { Id = relationshipId }, cancellationToken);
```

Comment:
As the audit log of a relationship might contain a few entries, maybe it's better to create a new query `GetPeerOfActiveIdentityInRelationship` which only returns the address of the peer. Just to save some unncessary data from being transferred from the database to the application.